docs/codeql/codeql-language-guides/basic-query-for-cpp-code.rst

```diff
@@ -134,14 +107,24 @@ To exclude ``if`` statements that have an ``else`` branch:
         block.getNumStmt() = 0 and
         not ifstmt.hasElse()
 
-#. Click **Run**.
+#. Re-run the query.
 
    There are now fewer results because ``if`` statements with an ``else`` branch are no longer reported.
 
-➤ `See this in the query console <https://lgtm.com/query/1899933116489579248/>`__
-
 Further reading
 ---------------
 
 .. include:: ../reusables/cpp-further-reading.rst
 .. include:: ../reusables/codeql-ref-tools-further-reading.rst
+
+.. Article-specific substitutions for the reusables used in docs/codeql/reusables/vs-code-basic-instructions
+
+.. |language-text| replace:: C/C++
+
+.. |language-code| replace:: ``cpp``
+
+.. |example-url| replace:: https://github.com/protocolbuffers/protobuf
```

Comment:
Question, mostly for  @jf205 : will `https://github.com/protocolbuffers/protobuf` work for everyone, or does this just work for us because of our GH privileges? If this doesn't work in general, I'm a bit concerned that people might bail out at this point, while they really haven't tried CodeQL.

Reply:
Great question. I think it's okay, because James approved adding example links, but it would be good to have confirmation that non-Hubbers have access.

Reply:
Hmmm, https://github.com/protocolbuffers/protobuf is public so will currently work for everyone (you can test it in an incognito window). There is a very small risk is that it may be made private in the future though.

@felicitymay, what's the general docs policy about linking to repos in examples?

Reply:
@jf205 That I believe, but can everyone also download the database?

Reply:
Yes. Users aren't required to be signed in when they download dbs from GitHub, so this will work.

Reply:
Great, thanks for clarifying. Then this LGTM.